types/data-api-client/tsconfig.json

```diff
@@ -6,6 +6,7 @@
         "noImplicitThis": true,
         "strictFunctionTypes": true,
         "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
Why do you need to change this?
Usually this is only required for specific cases for types consumption,
but not advised here.

Reply:
Ahh yeah right @peterblazejewicz I don't think it's even correct to edit this as this is not my type I will recheck again on my setup what caused this error